docs/src/terminology.md

```diff
@@ -67,6 +67,14 @@ A measure of the network confirmation for the [block](#block).
 
 A set of [validators](#validator) maintaining a single [ledger](#ledger).
 
+## cluster restart
+When there is an outage such that the whole [cluster](#cluster) stalls, human
```

Comment:
i'm not sure this helps much given as we similarly define neither "outage" nor "stall"